dockerfiles/Dockerfile.migraphx

```diff
@@ -22,5 +22,5 @@ RUN git clone --single-branch --branch ${ONNXRUNTIME_BRANCH} --recursive ${ONNXR
     /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh &&\
     cd onnxruntime  && pip install --upgrade pip &&\
     /bin/sh ./build.sh --allow_running_as_root --cmake_extra_defines ONNXRUNTIME_VERSION=`cat ./VERSION_NUMBER` --config Release --parallel \
-            --skip_tests --build_wheel --use_rocm --rocm_version=${ROCM_VERSION} --rocm_home /opt/rocm --use_migraphx &&\
+            --skip_tests --build_wheel --rocm_version=${ROCM_VERSION} --rocm_home /opt/rocm --use_migraphx &&\
```

Comment:
can `--rocm_version` and `--rocm_home` also be removed? here, and in build_args.py.

Reply:
removed